test/include/util/test_harness.h

```diff
@@ -0,0 +1,30 @@
+//
+// Created by pakhtar on 8/15/18.
+//
+
+#pragma once
+#include "gtest/gtest.h"
+#include "loggers/main_logger.h"
+#include "loggers/storage_logger.h"
+
+namespace terrier {
+namespace test {
```

Comment:
I think we're just doing the terrier namespace for the test subfolder, rather than terrier::test.